src/test/java/nl/knaw/dans/ingest/core/deposit/DepositLocationReaderImplTest.java

```diff
@@ -40,6 +40,7 @@ void readDepositLocation_should_call_with_correct_paths() throws Throwable {
         var config = new BaseConfiguration();
         config.setProperty("dataverse.sword-token", "token");
         config.setProperty("identifier.doi", "doi");
+        config.setProperty("creation.timestamp", "2022-10-01T00:03:04+03:00");
```

Comment:
Also test without this property and verify it throws an exception.

Reply:
Added test.